docs/docs/getting-started/01-intro.md

```diff
@@ -0,0 +1,20 @@
+---
+sidebar_position: 1
+---
+
+# Getting started
+
+:::caution
+
+The AICA framework is being distributed in a limited beta program. Installation, configuration and run-time instructions
+are subject to breaking changes in the near future. We will endeavour to keep this documentation up-to-date with the
+latest versions. In case of doubt, contact support@aica.tech for help.
```

Comment:
Did this email get setup already?

Reply:
Yes, we have `support@aica.tech` intended for existing users / support questions, and `contact@aica.tech` for all general and sales enquiries. I have tried to use the most appropriate address whenever it comes up